jerry-core/config.h

```diff
@@ -80,7 +85,7 @@
  * Also the option affects size of ECMA Object Model's data types.
  * In any case size of any of the types should not exceed CONFIG_MEM_POOL_CHUNK_SIZE.
  */
-#define CONFIG_ECMA_REFERENCE_COUNTER_WIDTH (10)
+#define CONFIG_ECMA_REFERENCE_COUNTER_WIDTH (9)
```

Comment:
I wouldn't decrease this, since even 1024 is not too big. Could we keep the original somehow?